components/ringcentral/actions/get-message/get-message.mjs

```diff
@@ -0,0 +1,43 @@
+import ringcentral from "../../ringcentral.app.mjs";
+
+export default {
+  key: "ringcentral-get-message",
+  name: "Get Message",
+  description: "Get message from the Message Store. See the API docs [here](https://developers.ringcentral.com/api-reference/Message-Store/readMessage)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ringcentral,
+    accountId: {
+      propDefinition: [
+        ringcentral,
+        "accountId",
+      ],
+    },
+    extensionId: {
+      type: "string",
+      label: "Extension ID",
+      description: "Extension ID of the message.",
+      propDefinition: [
+        ringcentral,
+        "extensionId",
+      ]
+    },
+    messageId: {
+      type: "string",
+      label: "Message ID",
+      description: "The ID of the message.",
+    },
+  },
+  async run({ $ }) {
+    const response = await this.ringcentral.getMessage(
+      this.accountId,
+      this.extensionId,
+      this.messageId,
+    );
```

Comment:
Passing along the `$` variable to the axios call helps us get more meaningful debugging info if an error is thrown. I would have `getMessage` accept an object including `$` as shown below, and then update the `getMessage` method in the app file as well.
```suggestion
    const response = await this.ringcentral.getMessage({
      $,
      accountId: this.accountId,
      extensionId: this.extensionId,
      messageId: this.messageId,
    });
```